buildkitd/buildkitd.go

```diff
@@ -424,10 +424,6 @@ func Start(ctx context.Context, console conslogging.ConsoleLogger, image, contai
 		envOpts["CACHE_SIZE_MB"] = strconv.FormatInt(int64(settings.CacheSizeMb), 10)
 	}
 
-	if settings.GitURLInsteadOf != "" {
```

Comment:
We should probably also remove `settings.GitURLInsteadOf` and also whatever flags or env vars are setting it on the other side.

Reply:
:facepalm: good call.